megamek/src/megamek/client/bot/princess/BehaviorSettingsFactory.java

```diff
@@ -229,7 +244,7 @@ public boolean saveBehaviorSettings(boolean includeTargets) {
         init(false);
         List<String> names;
         synchronized (behaviorMap) {
-            names = new ArrayList<String>(behaviorMap.keySet());
+            names = new ArrayList<>(behaviorMap.keySet());

```

Comment:
Yep, Java 7 added type inference support making diamond operators the preferred code style.  I have an inspection set up in my IDE to detect unnecessary explicit typing. 
http://docs.oracle.com/javase/tutorial/java/generics/genTypeInference.html